src/Compilers/Core/Portable/CommandLine/CommonCommandLineArguments.cs

```diff
@@ -143,6 +143,12 @@ public abstract class CommandLineArguments
         public ImmutableArray<CommandLineSourceFile> AdditionalFiles { get; internal set; }
 
         /// <summary>
+        /// An ordered list of .editorconfig files that are consumed by analyzers. The order is significant, and if more than one .editorconfig
```

Comment:
@sharwell, I'm not sure what you mean "compatible with this design". Can you clarify? If you're referring to packages providing their own .editorconfigs that apply to customer projects, that work is currently out of scope.